recipes/gfaviz/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "gfaviz" %}
+{% set version="1.0.0" %}
+{% set hash="4bc4327dd182812a9cfd53a39632bf41f461170e142df62ebbd0ba441239a6d4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x") }}
+
+source:
+  url: https://github.com/ggonnella/gfaviz/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - qt-main =5
+    - libgl-devel # [ linux ]
+  run:
+    - {{ pin_compatible('qt-main', max_pin='x.x.x') }}
+
+test:
+  commands:
+    - which gfaviz
+#    - gfaviz --help
```

Comment:
```suggestion
   - DISPLAY=:0.0 gfaviz --help
```

Reply:
thank for the clue

Reply:
Sure!
But now it fails to connect to it ...
I'll try to find other recipes that do something similar. 
You can also remove this test and test the produced binary locally.

Reply:
Yes, must be an headless ci. It was successful on my computer with `bioconda-utils build --docker --mulled-test --git-range master`.

Reply:
Any ideas how to make `xvfb-run` works with ci/cd?

Reply:
See how other recipes do it:
- https://github.com/conda-forge/mayavi-feedstock/blob/3c4d54956296e323577c0cb805fdc5ab28264bd8/recipe/build.sh#L7-L8
- https://github.com/search?q=org%3Aconda-forge%20xvfb&type=code

Reply:
Thanks @martin-g but it doesn't help.
The problem is that no `xvfb-run` nor `Xvfb` is found even if I use the right package according to [https://conda-metadata-app.streamlit.app](https://conda-metadata-app.streamlit.app)
Any idea ?